myapp/tests/test_scheduler.py

```diff
@@ -0,0 +1,16 @@
+from django.test import TestCase
+from ..scheduler import scheduler
+import threading
+
+
+
+class test_scheduler(TestCase):
+
+    def test_scheduler_thread(self):
+        one = threading.active_count()
+        job = scheduler()
+        job.start()
+        self.assertNotEqual(one, threading.active_count())
```

Comment:
be more specific and concrete please, how many threads you are going to create, and then use `assertEqual`

Reply:
three threads should be created , one for the main test , other two for scheduler , i can put three , but in the future make sure to increase that number

Reply:
> three threads should be created , one for the main test , other two for scheduler , i can put three , but in the future make sure to increase that number

I agree with you on this point